### PR TITLE
Add system preferences settings strings

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
These adds strings needed for #1680.

### CODE REVIEW

**Comment by lesleyjanenorton** [comment]
@flodolo  Sorry for the late ask here as I realize we are past all of the freezes. We were asked to get a refactor to the settings view in that requires these strings. Is it at all feasible/realistic that these could be translated in time for release?

**Comment by flodolo** [comment]
> Is it at all feasible/realistic that these could be translated in time for release?

I don't think it's realistic, mostly because of the amount of content we're asking community to translate these days is very large (we just exposed both the website and the add-on for Relay, Firefox 93 is approaching content freeze this week).

How feasible is to land the strings, and get the implementation behind some flags? Based on the number of translations done, we'll be able to chose if the flag can be flipped or not closer to release date.

File: translations/strings.yaml [hunk 1]
```diff
@@ -215,3 +215,7 @@ customDNSSettings:
 
 networkSettings:
   multihopTitle: Multi-hop tunnel
+
+systemPreferencesSettings:
+  systemPreferencesTitle: System preferences
```
Comment (flodolo) [inline translations/strings.yaml:220]: Out of curiosity, why "preferences"? Firefox is moving towards "settings", unified across platforms (macOS, Linux, Windows).

Comment (lesleyjanenorton) [reply translations/strings.yaml:220]: Thank you for the feedback! I'm not sure but maybe @chrisrooneys can add context.

**Pull request closed without merging**